src/geo/transform.js

```diff
@@ -1489,14 +1489,15 @@ class Transform {
         const cameraPixelsPerMeter = this.cameraPixelsPerMeter;
         const cameraPos = this._camera.position;
 
+        const windowScaleFactor = 1.0 / this.height;
         m = mat4.create();
+        mat4.fromScaling(m, [windowScaleFactor, windowScaleFactor, windowScaleFactor]);
```

Comment:
We could also apply windowScaleFactor like so, since we're already calling mat4.scale further down:
```
vec3.scale(metersPerPixel, metersPerPixel, windowScaleFactor)
```
I'll let you choose which one you think is more clear, no particular preference.

Reply:
I support any simplification, but there's a translation between the two, so you need to be very careful about transforms which don't commute.